packages/mermaid/src/diagrams/git/gitGraphTypes.ts

```diff
@@ -0,0 +1,133 @@
+import type { GitGraphDiagramConfig } from '../../config.type.js';
+import type { DiagramDBBase } from '../../diagram-api/types.js';
+
+export interface CommitType {
+  NORMAL: number;
+  REVERSE: number;
+  HIGHLIGHT: number;
+  MERGE: number;
+  CHERRY_PICK: number;
+}
```

Comment:
```suggestion
export interface CommitType {
  NORMAL: 0;
  REVERSE: 1;
  HIGHLIGHT: 2;
  MERGE: 3;
  CHERRY_PICK: 4;
}
```

Better to use the actual numbers in the type, so it's clearer. 
This commitType is defined twice, that should also be avoided.